client/src/components/Header/components/nav-links.tsx

```diff
@@ -497,19 +524,17 @@ class NavLinks extends Component<NavLinksProps, NavlinkStates> {
             </ul>
           </div>
         </li>
-        {username && (
-          <Fragment key='signout-frag'>
-            <li className='nav-line' key='sign-out'>
-              <button
-                className='nav-link nav-link-signout'
-                data-value='sign-out-button'
-                onClick={this.handleSignOutClick}
-                onKeyDown={this.handleLanguageMenuKeyDown}
-              >
-                {t('buttons.sign-out')}
-              </button>
-            </li>
-          </Fragment>
+        {currentUserName && (
+          <li className='nav-line' key='sign-out'>
+            <button
+              className='nav-link nav-link-signout'
+              data-value='sign-out-button'
+              onClick={this.handleSignOutClick}
+              onKeyDown={this.handleMenuKeyDown}
```

Comment:
Luckily, it was simple, it was using the wrong Event.

I had removed the weird Fragments, so I don't really mind adding it, to this PR.